node/README.md

```diff
@@ -11,19 +11,31 @@ Cross platform tasks are written in TypeScript.  It is the preferred way to writ
 
 [![NPM version][npm-lib-image]][npm-lib-url]
 
-Step by Step: [Create Task](https://docs.microsoft.com/en-us/azure/devops/extend/develop/add-build-task?view=vsts)  
+Step by Step: [Create Task](https://docs.microsoft.com/en-us/azure/devops/extend/develop/add-build-task?view=vsts)
 
 Documentation: [TypeScript API](docs/azure-pipelines-task-lib.md), [task JSON schema](https://aka.ms/vsts-tasks.schema.json)
 
 Guidance: [Finding Files](docs/findingfiles.md), [Minimum agent version](docs/minagent.md), [Proxy](docs/proxy.md), [Certificate](docs/cert.md)
 
+## Node 10 Upgrade Notice
+
+Azure DevOps is currently working to establish Node 10 as the new preferred runtime for tasks, upgrading from Node 6.
+Relevant work is happening in the releases/3.x branch and will be merged into master when we have successfully migrated the majority of our first party tasks.
+Any changes should be dual-commited to both `master` and `releases/3.x` until the merge into master occurs.
```

Comment:
Remove references to `releases/3.x` branch?

Reply:
Thank you for the review! Fixed it.